.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"bluedot_point_sdk","path":"/Users/longnguyen/Development/flutter/.pub-cache/git/PointSDK-Flutter-Plugin-cbeea1b714c8d2d00e274963e47a3c678f71204f/","native_build":true,"dependencies":[]}],"android":[{"name":"bluedot_point_sdk","path":"/Users/longnguyen/Development/flutter/.pub-cache/git/PointSDK-Flutter-Plugin-cbeea1b714c8d2d00e274963e47a3c678f71204f/","native_build":true,"dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"bluedot_point_sdk","dependencies":[]}],"date_created":"2022-09-01 18:37:12.361491","version":"3.0.5"}
```

Comment:
this file should be excluded from git

Reply:
and a few more folders. You can take a look at other ios/android projects to have some idea which should be gitignored.